setup.py

```diff
@@ -17,6 +17,10 @@
 exec(open(os.path.join(NAME, "about.py")).read())
 VERSION = __version__
 
+easy_process = "EasyProcess"
+if VERSION >= '0.2.5':
```

Comment:
This condition will always be true, as only newer versions have this `setup.py` change, no? Why not just use `EasyProcess>=0.3.0` directly below?

Reply:
agree specifying minimum version would be better.  I have to manually install EasyProcess==0.3.0 to fix breaking tests since I had previously installed a lower version via pyvirtualdisplay.

...I'm not sure if there would be regressions for lower versions of pyvirtualdisplay or python.

Reply:
> This condition will always be true, as only newer versions have this `setup.py` change, no? Why not just use `EasyProcess>=0.3.0` directly below?
Of course 0.3.0 is ok,  we can forcely user to install easyprocess >= 0.3.0. For the merge code, I just thought that there's no need to upgrade easyprocess to newest if it has been installed when user tried to upgrade virtualdisplay above 0.2.5， I've test it, it worked well.  But in anoter way, I perfer user to use new version of easyprocess, so forcely to install easyprocess >= 0.3.0 maybe a good choice.

Reply:
> agree specifying minimum version would be better. I have to manually install EasyProcess==0.3.0 to fix breaking tests since I had previously installed a lower version via pyvirtualdisplay.
> 
> ...I'm not sure if there would be regressions for lower versions of pyvirtualdisplay or python.


I tested that all the version of virtualdisplay can worked well if easyprocess < 0.3.0. But if your   env has install easyprocess >= 0.3.0, only 0.2.5 for virtualdisplay can work well.